mirage/views/osf-resource.ts

```diff
@@ -147,3 +147,61 @@ export function osfNestedResource<K extends keyof ModelRegistry>(
         }
     }
 }
+
+export function osfM2MRelationshipResource<K extends keyof ModelRegistry>(
```

Comment:
Minor: I think naming this `osfToManyRelationship` might be clearer -- these endpoints don't represent resources, just relationships. Also, it's closer to [json:api's terminology](https://jsonapi.org/format/1.1/#crud-updating-to-many-relationships).